mypy/semanal.py

```diff
@@ -4310,3 +4334,21 @@ def find_fixed_callable_return(expr: Expression) -> Optional[CallableType]:
             if isinstance(t.ret_type, CallableType):
                 return t.ret_type
     return None
+
+
+def unmark_any_as_explicit(t: Type) -> None:
```

Comment:
Yeah, naming things is hard - I went through several iterations of names trying to name this function. 

How about `unset_any_type_as_explicit`? Alternatively, we could try to make the function more generic and add a `bool` parameter `explicit`. Something like:
```py
def set_any_type_explicit_attribute(explicit: bool) -> None: 
    ...
```